tests/state/test_in_memory_key_value_store.py

```diff
@@ -0,0 +1,11 @@
+from winton_kafka_streams.state.in_memory_key_value_store import InMemoryKeyValueStore
+
+
+def test_inMemoryKeyValueStore():
```

Comment:
Pedantic, but could there be a test for the `get` method? Also, is there expected behaviour for `__getitem__` if the item does not exist?

Reply:
Good point. For now it follows the python dictionary standard, so it has a store['key'] accessor that throws and a .get('key') that returns None/a default value.

We might want to revisit this at some point in the future, but it's good enough for now.